aws/cloud/iam.rego

```diff
@@ -1737,4 +1737,88 @@ iam_deprecated_policies_metadata := {
     "Resource Type": "",
     "Policy Help URL": "",
     "Resource Help URL": "https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.Client.list_attached_user_policies"
-}
\ No newline at end of file
+}
+
+#
+# PR-AWS-CLD-IAM-048
+#
```

Comment:
@vatsalgit5118 no snapshot is being generated for PR-AWS-CLD-IAM-*

**Log:**
2023-08-31 23:24:17,156 - 	TEST_IAM_01:false
2023-08-31 23:24:17,156 - 	TEST_IAM_02:false
2023-08-31 23:24:17,156 - 	TEST_IAM_04:false
2023-08-31 23:24:17,156 - 	TEST_IAM_05:false
2023-08-31 23:24:17,156 - 	TEST_IAM_06:false
2023-08-31 23:24:17,156 - 	TEST_IAM_07:false

Reply:
@vatsalgit5118 snapshots are not getting generated.